File: LeetCode/26. Remove Duplicates from Sorted Array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        count = 1
        prev = nums[0]
        for i in range(len(nums)):
            if nums[i] != prev:
                nums[count] = nums[i]
                prev = nums[i]
                count += 1
        del nums[count:]

File: LeetCode/26. Remove Duplicates from Sorted Array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int count = 1;
        int prev = nums[0];
        for (int i = 0; i < nums.length; i++) {
            if (nums[i] != prev){
                nums[count] = nums[i];
                prev = nums[i];
                count++;
            }
        }
        return count;
    }
}
